.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,16 @@
 ## Purpose:
 What issue(s) does your PR address?
 
-## Results
-If applicable, briefly report your findings.
+## Strategy
+What was your strategy for this new or edited analysis?
+
+## Concerns/Questions for reviewers:
+What things should reviewers look out for?
 
 ## Pull Request Check List:
-* [ ] Run a linter
-* [ ] Set the seed (if applicable)
-* [ ] Comments and/or documentation up to date
-* [ ] Double check your paths
-* [ ] Check headline numbering
-* [ ] Spell check any Rmd file or md file
-* [ ] Restart R and run all notebooks fresh and save
+* [ ] [Sources are cited](https://github.com/AlexsLemonade/refinebio-examples/CONTRIBUTING.md#citing-sources-in-text)
+* [ ] Set the seed (if applicable).
+* [ ] Remove any manual numbering of sections.
+* [ ] Spell check any Rmd file or md file.
+* [ ] Run [r-lib/styler package](https://github.com/r-lib/styler)
+* [ ] Run [snakemake for rendering](https://github.com/AlexsLemonade/refinebio-examples/CONTRIBUTING.md#how-to-re-render-the-notebooks)
```

Comment:
Should we also add remove any instances of chunk naming here? It may not be necessary, what do you think?

Reply:
Sure!